pyproject.toml

```diff
@@ -44,17 +44,20 @@ scanpy = "*"
 lxml = "*"
 cssselect = "*"
 typing_extensions = "*"
-scipy = "<1.13.0"
+scipy = ">=1.0.0,<2.0.0"
 pyarrow = ">=14"
 semver = "^3.0.0"
 ruamel-yaml = "^0.17.21"
 pydantic = ">=2.0,<2.5"
 polars = ">=1.0.0,<2.0.0"
 importlib-resources = "^5.12.0"
-fsspec = "^2024.9.0"
-fastparquet = "^2023.8.0"
-graspologic = "^3.3.0"
+fsspec = "*"
+fastparquet = "*"
 plotly = "*"
+graspologic-native = "^1.2.4"
```

Comment:
Nota bene, we now only depend on graspologic-native directly, which is nice because it speeds up imports and lowers the amount of secondary dependencies we take on.